docs/requirements.txt

```
@@ -4,5 +4,5 @@ toolz
 cloudpickle
 dask
 sphinx
-dask_sphinx_theme
+dask-sphinx-theme>=1.3.3
```

We recently pushed out a new `dask-sphinx-theme` release, so I think we want: 

```suggestion
dask-sphinx-theme>=1.3.4
```